Add concept list for Bowern & Atkinson 2012 (bowernpny on lexibank)

tresoldi opened the pull request:
I forked the repository as I don't have write permission; I can branch there if you prefer.

This new concept list is passing check (`concepticon check Bowern-2012-348`) silently.

LinguList commented on concepticondata/concepticon.tsv:
```diff
@@ -3000,3 +3000,25 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3004	FORMER	Time	Coming before in time.	Property	
 3005	FAMILIAR	Emotions and values	Knowing somebody or something well.	Property	
 3006	SPACE (AVAILABLE)	Spatial relations	Available space to place things.	Person/Thing	
+3007	EUCALYPT	Agriculture and vegetation	Tree or wood for plants with capsule fruiting bodies belonging to closely related genera found across Australia.	
```
Please provide the basic word class information for all added concepts, that is person/thing, etc. The ontology is simple: person/thing is just  = noun, property is adjective, use other for things where you are not sure.

tresoldi replied:
Sorry, had planned to include it and forgot.

LinguList commented on concepticondata/concepticon.tsv:
```diff
@@ -3000,3 +3000,25 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
+3015	BLACK DUCK	Animals	Pacific black duck (Anas superciliosa).	
+3016	CROSS COUSIN	Kinship	Cousin from a parent's opposite-sex sibling (in relation to parallel cousin).	
+3017	PARALLEL COUSIN	Kinship	Cousin from a parent's same-sex sibling (in relation to cross cousin).	
+3018	FATHER'S FATHER	Kinship	Male grandparent of the father side (in relation to mother's father).	
```
Did you make sure we don't have this already in the form of Grandfather (of male)? We should have it, as the Chinese lists also make this distinction, i.e., grandfather from father's side vs. grandfather from mother's side.

tresoldi replied:
I've just checked: indeed it is there, as PATERNAL GRANDFATHER, MATERNAL GRANDMOTHER and MATERNAL GRANDFATHER (apparently, PATERNAL GRANDMOTHER is missing). Must have been a problem with my query, will correct it.

LinguList replied:
This is just what normally happens: We have list a) stating that it is "grandfather (paternal)", and then a new list b) stating that it is father's father. If you are not aware of list a), this can be accidentally be put into different concept sets, but now, that we spotted this, our algorithm for automatic linking will grow stronger and also find father's father if we compare against a new list.

LinguList commented on concepticondata/concepticon.tsv:
```diff
@@ -3000,3 +3000,25 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
+3023	GOANNA	Animals	Australian monitor lizard of the genus Varanus.	
+3024	QUOLL	Animals	Marsupial of the genus Dasyurus.	
+3025	SAND HILL	The physical world	Ecological community or xeric wildfire-maintained ecosystem (different from sand dune).	
+3026	URINATE	The body	To pass or discharge urine.	
```
We have defecate for "to shit", and we have "to pee" as well, I am sure, so you should link to this concept, instead of creating a new one.

tresoldi replied:
I will look for it, I did find it really strange for such a basic concept to be missing.

LinguList replied:
Its 592 PISS ;)

LinguList commented on concepticondata/conceptlists.tsv:
```diff
@@ -7,6 +7,7 @@ Ray-1895-26	Ray, Sidney H.	1895		26	ultra-stable,areal	English	Britich New Guina
 Backstrom-1992-210a	Backstrom, Peter C. and Radloff, Carla F.	1992	a	210	basic,areal	English	Pakistanian languages		Backstrom1992	Backstrom1992	This list was compiled for a sociolinguistic survey of language use in Pakistan. The list of 210 items is supposed to represent a standardized list used to investigate all languages in the field work survey. However, as can be seen when comparing the offi ...
 Backstrom-1992-210b	Backstrom, Peter C. and Radloff, Carla F.	1992	b	205	basic,areal	English	Balti dialects		Backstrom1992		This is the concrete list which was used for Balti dialects (Tibetan group of Sino-Tibetan) in the large survey of languages in Pakistan (see the [general list of 210 items used in this survey](:ref:Backstrom-1992-210a)). The list is missing five entries and shows also some  ...
 Bowern-2012-644	Bowern, Claire	2012		644	questionnaire	English	Tasmanian		Bowern2012a		This list supplements a study on Tasmanian languages based on various sources. It provides a linking between different concept labels found in various sources on Tasmanian languages. Due to the specificity of the vocabulary, we had to leave many concepts unlinked.		
+Bowern-2012-348	Bowern, Claire	2012		348	questionnaire	English	Pama-Nyungan	https://zenodo.org/record/1005671	Bowern2012b		This list supplements Bowern & Atkinson (2012) study on Pama-Nyungan languages; it partly overlaps with other concept lists by Bowern.		
```
what surprises me greatly is that this list has now 348 items while we were expecting 200, or less, according to the text, from which I linked the Bower-2012-200 (or similar) list originally. Does this mean that the list is a merger of the Swadesh-200 list plus the cultural terms which Bowern addressed in the Hunter Gatherer Project?

LinguList replied:
Also, if writing statements like "overlaps with", it is nice to give a link, such as:

```text
This lists supplements [Bowern and Atkinson's (2012)](:bib:Bowern2012) study and contains material from both Swadesh's [200-item list](:ref:Swadesh-1952-200) and the [questionnaire for hunter gatherer languages](:ref:Bowern-2011-204). 
```
So in a nutshell, we try to figure out how those concepts relate to each other, and I am convinced that this list now contains a merger of Swadesh 200 plus HG vocabulary, as otherwise you could not explain 348 items, although they only talk about less than 200 items in the paper, right? We may decide to just call this list Bowern-2017-348, to indicate that it is a later supplement to the paper, but we might also ask Claire directly, what she thinks.

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+2	Bowern-2012-348-2	1dl.incl	2637	WE TWO (INCLUSIVE)	
+3	Bowern-2012-348-3	3dl	2635	THEY TWO	
+4	Bowern-2012-348-4	Bloodwood	3007	EUCALYPT	
+5	Bowern-2012-348-5	Blue-Tongue Lizard	632	LIZARD	mapping to LIZARD as it's the only Lacertilia in the list
```
This kind of mapping may be allowed in rare cases, in this one, I'd say it is okay.

tresoldi replied:
Good, I almost opened an issue to ask about it. My impression was that, in general, it is better to have Concepticon as stable as possible, favoring corrections on concept lists, am I correct?

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+3	Bowern-2012-348-3	3dl	2635	THEY TWO	
+4	Bowern-2012-348-4	Bloodwood	3007	EUCALYPT	
+5	Bowern-2012-348-5	Blue-Tongue Lizard	632	LIZARD	mapping to LIZARD as it's the only Lacertilia in the list
+6	Bowern-2012-348-6	Brolga	1830	CRANE	mapping to CRANE as it's the only Gruidae in the list
```
okay, we have to many special names here, so I'd in the end rather decide to leave them unmapped or to add them as new concepts and add concept relations, such as Blue tongue lizard is narrower than lizard. You should check with the file on conceptrelations where we list this.

tresoldi replied:
Ok, I see. What I tried to do was map to the closest match when there was a single entry (like BLUE-TONGUE LIZARD > LIZARD) and map one of the items to its closest match in case of multiple entries, adding the other ones.

I'll check the concept relations.

LinguList replied:
Yes, I am afraid it is needed. It is already an almost uncontrolable task with more than 3000 concept sets. The more important that we try to stay as put as closely to strict definitions of concepts, which also revokes what I said earlier about the lizard. What I also do at times, and I find it justified, is just empty the cell and avoid the definition. We don't need to link everything, at times, we can allow to leave things unlinked, if we find they won't be important for cross-linguistic comparison.

tresoldi replied:
In this case it is actually easier: there are many concepts, for example, that really only make sense to Pama-Nyungan (and perhaps other Australian languages) and there is really no need to include in the large Concepticon-scope. I will take a new look at everything and maybe think of solutions to make this overlap identification easier.

LinguList replied:
yes, you know, just leave them out. We have the NA class, which all concepts will be assigned to that we can't map, and we can later still get back an re-link. But it's good that you learned the general drill of adding a new concept, adding a concept-relation, and making the intro of a concept list. These are all crucial for working with lexibank (but also concepticon/cldf in general).

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+24	Bowern-2012-348-24	all	98	ALL	
+25	Bowern-2012-348-25	alone	1964	ALONE	
+26	Bowern-2012-348-26	and	1577	AND	
+27	Bowern-2012-348-27	angry	1826	ANGRY	check if it shouldn't be BE ANGRY
```
angry is okay.

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+29	Bowern-2012-348-29	ant	587	ANT	
+30	Bowern-2012-348-30	armpit	1886	ARMPIT	
+31	Bowern-2012-348-31	ash	646	ASH	
+32	Bowern-2012-348-32	ask	1309	ASK (INQUIRE)	check if it should be ASK (REQUEST)
```
ask-request is normally not used, or it is made explicit, so ask inquire is good!

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+47	Bowern-2012-348-47	black duck	3015	BLACK DUCK	
+48	Bowern-2012-348-48	blind	1653	BLIND	
+49	Bowern-2012-348-49	blood	946	BLOOD	
+50	Bowern-2012-348-50	blow	175	BLOW (OF WIND)	needs to be confirmed
```
we often have these cases, and I remember that there was confusion in the original dataset, as they did not always distinguish between what "blow" could mean. We may consider: a) adding a meta-concetp (blow (with mouth or of wind)), but this is tedious, or we make the bold decision to assume it is b) blow (of wind). I'd be inclined to go with b), as this is a very data-specific problem.

tresoldi replied:
Good. I am collecting the doubts to ask experts later on, but I tried to find out or infer the concept for these cases by looking at the data or with some google-fu. "blow", if I remember correctly, was one of the cases I was able to find, even though with a low confidence level.

LinguList replied:
In chirila, the database which will supersedes this dataset at some point in the future, there are dubious cases, I remember, but I also think that Claire wanted to fix them. I am fine with linking to "blow" now, as it was surely intended to mean "blow of wind", neither the blow as a beat or the blow of the mouth or the trumpet.

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+55	Bowern-2012-348-55	boomerang/throwing stick	135	BOOMERANG	
+56	Bowern-2012-348-56	bottom grinding stone	3028	BOTTOM GRINDSTONE	
+57	Bowern-2012-348-57	boy	1366	BOY	
+58	Bowern-2012-348-58	break (intr)	680	BREAK (DESTROY OR GET DESTROYED)	
```
Don't we have an intransitive version of "BREAK"? It seems not, but this concept justifies that we should add one AND that we need to add a conceprelation stating that BREAK (BREAKING) (I tend to use the ING-form to indicate intransitive) is narrower than 680.

tresoldi replied:
I might have missed it, but the only other one I found is BRAKE (CLEAVE). There is also BREAK (OF ROPE) that I would take as normally intransitive, but seems to specific. Will take a look here too.

LinguList replied:
No, it's not there yet, I checked, so we will need:

* BREAK (BREAKING)

as the intrasitive form. So far, we don't have a transitive form so we can leave that out.

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+76	Bowern-2012-348-76	cloud	1489	CLOUD	
+77	Bowern-2012-348-77	cold	1287	COLD	
+78	Bowern-2012-348-78	come	1446	COME	
+79	Bowern-2012-348-79	cook	1100	COOK (SOMETHING)	
```
Again an insecurity, as it could be the cook, but I think it is rather unlikely for those questionnaires, so the linking is correct, I'd do the same.

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+94	Bowern-2012-348-94	die/be dead	1494	DIE	
+95	Bowern-2012-348-95	dig	1418	DIG	
+96	Bowern-2012-348-96	digging stick	339	DIGGING STICK	
+97	Bowern-2012-348-97	dingo/wolf	2461	WOLF-LIKE ANIMAL	
```
this is a good example for how we handle uncertainty in conceptrelations. I remember now: this is the hunter-gatherer list for sure, as I added the wolf-like animal for this purpose.

tresoldi replied:
Ok, will take it as a "starting point".

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+181	Bowern-2012-348-181	left	244	LEFT	
+182	Bowern-2012-348-182	leg	1297	LEG	
+183	Bowern-2012-348-183	lie down	215	LIE DOWN	
+184	Bowern-2012-348-184	light	630	LIGHT (RADIATION)	confirm that it is not a different LIGHT concept
```
difficult but probably really the noun reading, unless you find "heavy" in the list as well?

tresoldi replied:
"HEAVY" is there, good to see it is the reasoning I had. In fact, now I come to think that it is probably "light" as "not heavy".

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+298	Bowern-2012-348-298	think	2271	THINK	
+299	Bowern-2012-348-299	thirsty	847	THIRSTY	
+300	Bowern-2012-348-300	this	1214	THIS	
+301	Bowern-2012-348-301	thou	2844	YOU (FORMAL ADDRESS)	Or is it plural?
```
this MUST be THOU 2nd person singular!

tresoldi replied:
I had it coded that way, but there are English glosses for both "you" and "thou" (apparently they are singular and plural). As I did not find a "YOU (PLURAL)" (only as the possessive or as a dual), I tried to do a concept relation, but was probably misguided by thinking about the etymology of the English glosses and not the concepts.

Still, I am puzzled: no concept for plural you was added so far?

LinguList commented on concepticondata/conceptlists/Bowern-2012-348.tsv:
```diff
@@ -0,0 +1,349 @@
+310	Bowern-2012-348-310	tomorrow	1329	TOMORROW	
+311	Bowern-2012-348-311	tongue	1205	TONGUE	
+312	Bowern-2012-348-312	tooth	1380	TOOTH	
+313	Bowern-2012-348-313	top grinding stone	3027	TOP GRINDSTONE	
```
Did you add the concept TOP GRINDSTONE? If so, it is probably narrower as relation that GRINDSTONE, right? This should be added.

tresoldi replied:
I did, the conceptlist has both TOP GRINDSTONE and BOTTOM GRINDSTONE. I was about the code the top grindstone as GRINDSTONE and the bottom one as either STONE or (adding the concept of) BOTTOM GRINDSTONE, don't exactly recalled why I decided to do it this way.

Will check the relation.

LinguList commented on concepticondata/references/references.bib:
```diff
@@ -1,6 +1,15 @@
 % This file was created with JabRef 2.10.
 % Encoding: UTF8
 
+@misc{Bowern2012b,
```
If this is the dataset, it was published in 2017, not in 2012. As we already saw that the datasets itself contains HG data plus Swadesh, it is essential to distinguish it from the 2012 publication, at least I am inclined to think about it like this.

tresoldi replied:
I agree with you, I went with 2012 because it is the date of the reference. It is almost an ontological question: the conceptlist should be considered a part of the dataset, derived from it, or a new work?

But as said I completely agree with you that 2017 would be better, I myself had some moments of confusion due to the Bowern-2012 already there. WIll happily change it.

tresoldi commented:
I've updated most things, but I was very conservative in the concepts that I had created and removed. Just please tell me which ones exactly I should remove from the global list (so that they will not be mapped in the `bowernpny` dataset); so far, all concepts are mapped.

LinguList commented on concepticondata/conceptlists/Bowern-2017-348.tsv:
```diff
@@ -7,19 +7,19 @@ NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	COMMENTS
 7	Bowern-2012-348-7	Carpet snake	1563	PYTHON	mapping to PYTHON as it's the only Pythonidae in the list
 8	Bowern-2012-348-8	Crow	913	CROW	
-9	Bowern-2012-348-9	Death adder	3008	DEATH ADDER	
+9	Bowern-2012-348-9	Death adder	2654	VIPER	Mapping to VIPER as closer concept for "venomous snake"
```
fine, I agree. This comment can be removed.

LinguList commented on concepticondata/conceptlists/Bowern-2017-348.tsv:
```diff
@@ -44,8 +44,8 @@ NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	COMMENTS
+47	Bowern-2012-348-47	black duck	3014	BLACK DUCK	
 48	Bowern-2012-348-48	blind	1653	BLIND	
 49	Bowern-2012-348-49	blood	946	BLOOD	
 50	Bowern-2012-348-50	blow	175	BLOW (OF WIND)	needs to be confirmed
```
let's remove the comment and accept it as "blow (of wind)".

LinguList commented on concepticondata/conceptlists/Bowern-2017-348.tsv:
```diff
@@ -84,7 +84,7 @@ NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	COMMENTS
 84	Bowern-2012-348-84	crawl	96	CRAWL	
 85	Bowern-2012-348-85	creek	990	CREEK (TIDAL)	
-86	Bowern-2012-348-86	cross cousin	3016	CROSS COUSIN	
+86	Bowern-2012-348-86	cross cousin	1643	COUSIN	mapping to COUSIN as there is not other COUSIN item in this conceptlist
```
yes, I agree with this logic.

LinguList commented on concepticondata/conceptlists/Bowern-2017-348.tsv:
```diff
@@ -311,12 +311,12 @@ NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	COMMENTS
 311	Bowern-2012-348-311	tongue	1205	TONGUE	
 312	Bowern-2012-348-312	tooth	1380	TOOTH	
-313	Bowern-2012-348-313	top grinding stone	3027	TOP GRINDSTONE	
+313	Bowern-2012-348-313	top grinding stone	2459	GRINDSTONE	
```
sorry for my ignorance, but if there's a bottom grindstone, why would we not keep it as a concept? Mapping to a broader concept in this case seems unnecessary, since you can also handled it via the conceptrelations.tsv

tresoldi replied:
I am really in doubt about this; what I did was look for translations in some languages and try to infer what would be better as "default" (top or bottom grindstone). It *seems* that it is more common to call the top one as "grindstone", with second something like "stone/plane/base/wheel" plus some attribute, but it is really a wild guess. To be clear, I understand it refers to this:

![grindstone](http://www.yourlifechoices.com.au/assets/images/_thumbs550/ancient_grindstone_used_to_crush_grain_website.jpg)

My suggestion would now be to add no narrower grindstone concept, map Bowern's "top grindstone" to GRINDSTONE, and leave "bottom grindstone" unmapped. What do you think?

LinguList replied:
well, assuming that if ever we are going to make a study on grindstones in the world, we will more likely include the normal grindstone in our concept list than the top one. Yes, leave it unmapped, so we keep the order, and if people are unhappy, they can refine in twenty years, when grindstone linguistics becomes a hype...

tresoldi commented:
Changes in the last commit:

- I changed the prefix from "Bowern-2012-348" to "Bowern-2017-348"
- Made all the changes we agreed (grindstone, etc.)
- Considering other concept lists (especially Robinson's Alor-Pantar, which I will commit soon), I changed "quoll" and "goanna" to broader concepts "native cat" and "monitor lizard"; the first is due to the custom of calling all weasel-like animals as cat (native cat, civet cat, etc.) -- which is what we were discussing on David's blog and might have happened even in Latin; the second should be uncontroversial (I looked at some translations on Wiktionary, it seems that there is a general lexeme division for lizard/monitor lizard, but otherwise monitor lizards are distinguished by attributes).

LinguList commented:
sorry, somehow my last review was not submitted, I thought it was already readable, but please have a look. I am really concerned if we start adding "bottom grindstone" but not "top grindstone" and link "top grindstone" to "grindstone", better leave it unlinked, or add concept relations, as we try to avoid fuzzy mappings, we really need to be as exact as possible. I'll give some more comments now.

LinguList commented:
> Considering other concept lists (especially Robinson's Alor-Pantar, which I will commit soon), I changed "quoll" and "goanna" to broader concepts "native cat" and "monitor lizard"; the first is due to the custom of calling all weasel-like animals as cat (native cat, civet cat, etc.) -- which is what we were discussing on David's blog and might have happened even in Latin; the second should be uncontroversial (I looked at some translations on Wiktionary, it seems that there is a general lexeme division for lizard/monitor lizard, but otherwise monitor lizards are distinguished by attributes).

I don't know the other list, but please keep this thing in mind: only in the rarest cases do we link to broader concepts, unless they are commonly used in all kinds of questionnaires. Rather leave things unlinked, or add new definitions and use the conceptrelations.tsv in order to make sure that concepts are comparable. 

Somehow, my original review was not submitted, so I am repeating it here:

Unmappings to be done:

- [ ] unmapp eucalypt / bloodwood it is too fuzzy and doesn't occur in other lists
- [ ] unmapp blue-tongue lizard or make new concept plus concept relations, we don't want a specific kind of lizard to be linked to a general lizard
- [ ] unmapp brolga
- [ ] unmapp carpet snake
- [ ] unmapp death adder (it might confuse our automatic mapping)
- [ ] unmapp bottom grinding stone or add a new concept and add a conceptrelations.tsv (but then add top grinding stone as well)
- [ ] unmapp cross cousing (it's too shaky for me)

Uncheckings (remove comment, if submitted, we don't want to have comments in the file)

- [ ] ask (it's fine)
- [ ] angry is fine
- [ ] blow is fine
- [ ] light is fine

Correct errors:

THOU needs to be linked to concepticon THOU. I think this is almost it, but the THOU is important, it is just our concepticon THOU, not YOU. I had to change some 40 lists in the past, as I had not paid attention to this, since then, I'm sensible and usually check this at first...

Resolve conflict: bibliography

tresoldi commented:
Changes in last commit (and sorry for bothering you so much, with lingpy to release and all):

- Upstream was merged
- I removed the bloodwood -> EUCALYPT mapping in Bowern-2017, but kept the new concept in Concepticon, as I already have it in the Robinson list to commit next (please tell me if it is too specific and it should be removed, mapping to TREE or something in Robinson list)
- Mapped "blue-tongued lizard" to "BLUE TONGUED SKINK" (correct English name in biological literature), adding the concept as a "narrower" (hyponim) of LIZARD
- Unmapped "Brolga", "Carpet snake", "Death adder", and "cross cousin"
- Added TOP GRINDSTONE and BOTTOM GRINDSTONE, mapping them in Bowern and adding them as "narrowers" of GRINDSTONE in concept relations
- Removed comments in ASK, ANGRY, and LIGHT
- Mapped "thou" to THOU and "you.SG" to YOU
- Removed all comments and the COMMENT column

tresoldi commented:
Some more concept relations are probably needed, such as for WHITE/BLACK COCKATOO, BLACK DUCK, etc.

LinguList commented:
excellent, sorry for torturing you, it's the first time I review such a long list. and the first time is always difficult, will be easier from now on ;)

LinguList commented:
we'll ignore the cockatoo etc. for now. Later generations may try to make sure that black ducks are perceived as ducks.

tresoldi commented:
I've seen that there is some preliminary mapping to Wordnet, maybe in the future a tool could suggest relationships for us to manually check (btw, the cockatoos and the ducks are there).

LinguList commented:
we have been doing this with babelnet in the past: I have just checked the things I found and then added the metadata subsequently, but I think we need a fully funded project for this kind of endeavour.
